cri.go

```diff
@@ -78,6 +80,12 @@ func initCRIService(ic *plugin.InitContext) (interface{}, error) {
 		return nil, errors.Wrap(err, "failed to set glog level")
 	}
 
+	if c.DisableHTTP2Client {
```

Comment:
If we just want this to affect image pulls, do we need to globally modify `http.DefaultClient`?

Reply:
No we can just construct a new blank http.Client in place of this, this was just the client we/cri was using in the first case. Not a fan of modifying DefaultClient either but wasn't sure if this was the default for a reason.